doc/nrf/ug_thread_certification.rst

```diff
@@ -0,0 +1,109 @@
+.. _ug_thread_cert:
+
+Thread certification
+####################
+
+Thread Group requires a certification for devices using the Thread protocol.
+You can find below the particularities of Nordic Semiconductor devices using nRF Connect SDK when it comes to the Thread certification.
+
+For general details about the certification process, check the `Thread Group's certification information`_.
+
+Thread certification options
+****************************
+
+Depending on your development approach, you have the following certification options when using Nordic Semiconductor devices.
+
+.. contents::
+	:local:
+	:depth: 2
+
+
+Certification by inheritance without modifications to binaries
+==============================================================
+
+If you are developing a Thread product with nRF Connect SDK, you can apply for certifying this product by inheritance as an official Thread certified product.
+This is possible for products that are based on systems from Nordic (both stack and device) that were previously certified by Thread Group, as long as you have not modified the initial functionality that was tested by Thread Group for certification.
```

Comment:
The sentence is a bit long with a lot of 'that' clauses.
Maybe split into two sentences around these lines:
```suggestion
This is possible because the Thread stack and devices delivered by Nordic Semiconductor are already certified by Thread Group.
As long as you do not modify their initial functionality, your product is eligible for certification by inheritance.
```

Reply:
fixed